backend-models/service-tests/fileedservice-tests.txt

```diff
@@ -0,0 +1,127 @@
+Fileedservice.php
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+Test to deleting a file (line 58)
+--------------------------------------------------------------------------------------------------------------------------------------------
+
+*********
+* pre-req:
+*********
+(checklogin() && $hasAccess) { IS &&
+(strcmp($opt, "DELFILE") === 0 && (hasAccess($userid, $cid, 'w') || isSuperUser($userid))) IS TRUE
+
+
+*********
+* Send parameters inputs to fileedservice.php:
+*********
+
+send
+{
+$cid = 1234;
+$opt = "DELFILE";
+$coursevers = "test";
+$filename = "test.txt";
+$kind = 1;
+$fid = 1;
+$contents = "";
+}
+
+*********
+* Call the file deletion function:
+*********
+
+$result = delete_file($cid, $opt, $coursevers, $fid, $filename, $kind, $contents);
+
+*********
+* Verify that the file has been deleted:
+*********
+
+if ($result === true) {
+    echo "File deleted successfully.";
+} else {
+    echo "Error deleting file: " . $result;
+}
+
+*********
+* Expected output:
+*********
+
+true
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+Test retrieving a file path (line 218)
+--------------------------------------------------------------------------------------------------------------------------------------------
+
+*********
+* pre-req:
+*********
+
+(checklogin() && $hasAccess) { IS TRUE
+
+
+*********
+* Set up testdata:
+*********
+
+$fid = 1;
+$expectedPath = "/path/to/file";
+
+
+*********
+* Query the database:
+*********
+
+$query = $pdo->prepare("SELECT path from fileLink WHERE fileid = :fid");
+        $query->bindParam(':fid', $fid);
+        $result = $query->execute();
+        if($row = $query->fetch(PDO::FETCH_ASSOC)){
+            $actualPath = $row['path'];
+        }
+
+*********
+* Check that the actual path is equal to the expected path:
+*********
+
+$this->assertEquals($expectedPath, $actualPath);
+
+*********
+* Expected output:
```

Comment:
Change to:
*****************
 Service output
*****************

** Gather service output *
Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
this is the expected output for the micro service.